src/Model/ModelTrait.php

```diff
@@ -37,8 +39,9 @@ trait ModelTrait
      * @param array $data
      * @param bool $update
      * @param bool $replaceCollections
+     * @throws \ReflectionException|ModelException
      */
-    protected function loadData(array $data, $update = false, $replaceCollections = false)
+    protected function loadData(array $data, bool $update = false, bool $replaceCollections = false)
```

Comment:
`$replaceCollections` should be an array. Can you update the typehint, docblock and set the default to `[]` please?

Reply:
Ah I understand where my confusion was, I thought $replaceCollections was a bool because the default was false. Fixed now